packages/javascript/src/index.ts

```diff
@@ -346,7 +346,9 @@ export default class Appsignal implements JSClient {
     try {
       return await fn()
     } catch (e) {
-      await this.sendError(e, tagsOrFn, namespace)
+      if (e instanceof Error) {
```

Comment:
Noemi made it so that ErrorEvents are also accepted in PR #632. I think this would break reporting those.
Either we remove this if-statement because we already check in `send` if it's an error object or something different (if removing it is allowed by TypeScript), or it also checks for ErrorEvents here.